<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>501</width>
      <height>361</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabPrice">
     <attribute name="title">
      <string>가격수집</string>
     </attribute>
     <widget class="QDateEdit" name="qfrdate">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>10</y>
        <width>128</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="qtodate">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>40</y>
        <width>128</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>40</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>종료일</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>10</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>시작일</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="LogPrice">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>460</width>
        <height>250</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPortfolio">
     <attribute name="title">
      <string>포트폴리오 수집</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>10</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>기준일</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="qstddate">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>10</y>
        <width>128</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="LogPtf">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>460</width>
        <height>250</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="cboURL">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>40</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>40</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExt">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>501</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rdoBond">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>12</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>채권</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>분류</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoStock">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>12</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>주식</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTickerIN">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTickerNotIN">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkIN">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>개별</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkNotIN">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>71</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>제외</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="pbar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>22</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
